packages/beacon-node/test/unit/network/peers/priorization.test.ts

```diff
@@ -14,6 +14,7 @@ import {RequestedSubnet} from "../../../../src/network/peers/utils/index.js";
 
 type Result = ReturnType<typeof prioritizePeers>;
 
+// eslint-disable-next-line vitest/valid-describe-callback
 describe("network / peers / priorization", async () => {
```

Comment:
why is this eslint disable required?

Reply:
`vitest/recommended` eslint rules suggest to have no `async` for `describe` blocks and move such code into hooks.

Reply:
Silly question, do you think we can/should put the setup in a hook?  Seems like it can but am guessing you have a reason to go this route instead.

Reply:
There are multiple ways 

1. Do setup in test level hooks
2. Create a file for config level `setupFile` that execute before every test context. 
3. Create a file for config level `globalSetup` that execute before a test run. 

Te basic idea is any `async` code usually evolve some resources, so calling it in `beforeAll` or `beforeEach` and then clearing resources in `afterAll` or `afterEach` is preferred. If you do on describe level, there is more chances you forget to clear the resources.